routes/auth/auth.routes.js

```diff
@@ -0,0 +1,141 @@
+const User = require('../../models/User.model');
+
+const router = require('express').Router();
+const mongoose = require('mongoose');
+
+const passport = require('passport');
+const LocalStrategy = require('passport-local');
+
+const bcryptjs = require('bcryptjs');
+const saltRounds = 10;
+
+// require auth middleware
+const { isLoggedIn, isLoggedOut } = require('../../middleware/middleware.js');
+
+// https://github.com/howardmann/authentication
+passport.use(
+  'local-login',
+  new LocalStrategy(
```

Comment:
I personally would pull out these things. i.e. have a constant for the new LocalStrategy and a named function for the callback you pass into it. this way the callback function starting at line 25 is 2 levels less indented :)